uxarray/grid/grid.py

```diff
@@ -2446,3 +2459,81 @@ def get_faces_between_latitudes(self, lats: Tuple[float, float]):
 
         """
         return faces_within_lat_bounds(lats, self.face_bounds_lat.values)
+
+    def get_faces_containing_point(self, point, error_tolerance=ERROR_TOLERANCE):
```

Comment:
```suggestion
    def get_faces_containing_point(self, point, tolerance =ERROR_TOLERANCE):
```

Small suggestion to be consistent with how Xarray names their user-defined tolerance.